components/frontend/src/header_footer/Menubar.test.js

```diff
@@ -89,9 +89,9 @@ it('shows the view panel on menu item click', () => {
     expect(screen.getAllByText(/Hello/).length).toBe(1)
 });
 
-it('shows the view panel on enter', async () => {
+it('shows the view panel on space', async () => {
```

Comment:
New widget?

Reply:
No, toggling menu items should be done by spacebar if I understand the recommendations correctly (e.g. https://webaim.org/techniques/keyboard/). Although this is opening the settings panel, so maybe this is better done with enter, the linked guideline does not contain an entry on expanding/collapsing things.